system-tests/lib/cre/don/node/node.go

```diff
@@ -130,10 +131,6 @@ func FindManyWithLabel(nodes []*types.NodeMetadata, wantedLabel *types.Label, la
 		}
 	}
 
-	if len(foundNodes) == 0 {
-		return nil, fmt.Errorf("node with label %s=%s not found", wantedLabel.Key, wantedLabel.Value)
-	}
-
```

Comment:
this is a potentially sneakly-breaking change, but I think it makes sense to allow it to return `0` matches as a valid result